igvc_navigation/launch/robot_state_publisher.launch

```diff
@@ -3,6 +3,6 @@
     This file launches the state_publisher node that broadcasts the tf structure of the robot.
     -->
 <launch>
-    <param name="robot_description" command="cat $(find igvc_description)/urdf/jessi.urdf"/>
+    <param name="robot_description" command="$(find xacro)/xacro $(find igvc_description)/urdf/jessii.urdf.xacro gpu:=false" />
```

Comment:
`gpu` is only for gazebo and not used for the real robot. It's confusing to other people looking at this to see a `gpu` parameter.

Reply:
removed